meilisearch/task.py

```diff
@@ -0,0 +1,116 @@
+from time import sleep
+from datetime import datetime
+from typing import Any, Dict, List, Optional
+
+from meilisearch._httprequests import HttpRequests
+from meilisearch.config import Config
+from meilisearch.errors import MeiliSearchTimeoutError
+
+class Task:
+    """
+    Tasks routes wrapper.
+
+    Task class gives access to all Task routes and child routes via an Index (inherited).
```

Comment:
I may be misunderstanding this description, but I think it should read `Task class gives access to all Task routes` because it is not specific to an `Index`. In cases where it is for an `Index` it is based on the `id` of the index and not inherited from the `Index` class.